magento/magento2
# Fix typo on product details causing Validation error.

### Description


Fixes a typo on the aria-labeleledby attribute on the product page causing a W3C Validation error.

### Fixed Issues (if relevant)


### Manual testing scenarios

1. Navigate to the view page of any product.
2. Check the code on the W3C html validator.
3. Currently product pages return a "Attribute aria-labeledby not allowed on element div at this point." error as there is a typo in  app/code/Magento/Catalog/view/frontend/templates/product/view/details.phtml

### Contribution checklist
